pyproject.toml

```diff
@@ -168,6 +168,34 @@ ignore = [
     "RUF005",  # unpack-instead-of-concatenating-to-collection-literal -- it's not clearly faster.
 ]
 
+[tool.ruff.extend-per-file-ignores]
+"setup.py" = ["INP001"]  # Part of configuration, not a package.
+"__init__.py" = ["E402", "F401", "F403"]
+"astropy/io/registry/compat.py" = ["F822"]
+"astropy/modeling/models.py" = ["F401", "F403", "F405"]
+"astropy/utils/decorators.py" = [
+    "D214", "D215",  # keep Examples section indented.
+    "D411",  # sphinx treats spaced example sections as real sections
+]
+"test_*.py" = [
+    "B018",  # UselessExpression
```

Comment:
We don't need this anymore because eb61cc747808dc5f78633c078c4ab2270af02895 added `noqa` instructions to individual lines.

Reply:
Unfortunately we do still need this as noqa were only added to non-test files.